packages/lwc-language-server/src/__tests__/component-indexer.test.ts

```diff
@@ -125,6 +126,54 @@ describe('ComponentIndexer', () => {
                 expect(componentIndexer.tags.size).toBe(8);
             });
         });
+
+        describe('typescript path mapping', () => {
+            const expectedComponents: string[] = [
+                'force-app/main/default/lwc/*/hello_world',
+                'force-app/main/default/lwc/*/import_relative',
+                'force-app/main/default/lwc/*/index',
+                'force-app/main/default/lwc/*/lightning_datatable_example',
+                'force-app/main/default/lwc/*/lightning_tree_example',
+                'force-app/main/default/lwc/*/todo_item',
+                'force-app/main/default/lwc/*/todo',
+                'force-app/main/default/lwc/*/typescript',
+                'force-app/main/default/lwc/*/utils',
+                'utils/meta/lwc/*/todo_util',
+                'utils/meta/lwc/*/todo_utils',
```

Comment:
This is a sample of how the `paths` mapping will look like, it includes both `.js` and `.ts` files so TypeScript can resolve all LWC module locations.